### ISSUE TITLE
How to downgrade to older version

### ISSUE THREAD

**Issue by squadramunter** [lhark/truckersmp-cli#35]
Hello I have a question. How to downgrade to an older version of ETS2

I want to play TruckersMP but it is not compatible with the newest update.

Regards,

SquadraMunter

**Comment by WantedFreak** [comment]
In Steam Library, right click on ETS2, properties, choose BETA tab. Select the latest version from the list. 
https://i.imgur.com/VL4OJVg.png
When selected what version you want it will start to download the downgraded version.

**Comment by Lucki** [comment]
Append `-beta <name_of_beta>` after the steamid:
https://github.com/lhark/truckersmp-cli/blob/8e95826bacab03044073fdba6c30fb4ebef97d36/truckersmp-cli#L338

~~~ diff
- +app_update "$steamid" validate \ 
+ +app_update "$steamid" -beta <name_of_beta> validate \ 
~~~

**Comment by kakurasan** [comment]
`--beta VERSION` option (e.g. `--beta temporary_1_36`) has been implemented in 8aa88ca.
